src/config/entities/__tests__/configuration.ts

```diff
@@ -21,35 +21,6 @@ export default (): ReturnType<typeof configuration> => ({
   balances: {
     balancesTtlSeconds: faker.number.int(),
     providers: {
-      valk: {
-        baseUri: faker.internet.url({ appendSlash: false }),
-        apiKey: faker.string.hexadecimal({ length: 32 }),
-        chains: {
-          1: { chainName: faker.string.sample() },
-          10: { chainName: faker.string.sample() },
-          100: { chainName: faker.string.sample() },
-          1101: { chainName: faker.string.sample() },
-          1313161554: { chainName: faker.string.sample() },
-          137: { chainName: faker.string.sample() },
-          324: { chainName: faker.string.sample() },
-          42161: { chainName: faker.string.sample() },
-          42220: { chainName: faker.string.sample() },
-          43114: { chainName: faker.string.sample() },
-          56: { chainName: faker.string.sample() },
-          8453: { chainName: faker.string.sample() },
-        },
-        currencies: Array.from(
-          new Set([
-            ...Array.from(
-              { length: faker.number.int({ min: 2, max: 5 }) },
-              () => faker.finance.currencyCode(),
-            ),
-            'ETH',
-            'EUR',
-            'USD',
-          ]),
-        ),
-      },
       zerion: {
```

Comment:
Need will still keep this level of nesting? Although, with the manager, I suppose we do.

Reply:
Good point, after adding the TransactionApi to the manager I will review this configuration format 👍 Thanks!